### PR TITLE
Simplify dropdown DOM an make styling consistent with text fields

### LINKED ISSUE #228: Dropdown height is not the same as text field height
image

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Fixes #228

### CODE REVIEW

File: src/stories/FormFields.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Meta, Props, Story, Preview } from '@storybook/addon-docs/blocks';
+import { action } from "@storybook/addon-actions";
+import { withKnobs, object, boolean, text, select } from '@storybook/addon-knobs';
+import { withInfo } from 'storybook-addon-vue-info'
+import TextField from '../components/TextField.vue';
+import ChecDropdown from '../components/ChecDropdown.vue';
+
+<Meta title="Form fields" component={TextField} />
```
Comment (robbieaverill) [inline src/stories/FormFields.stories.mdx:8]: Did you deliberately put this into its own "Overview" story? If so, all good.

The text field's label says "dropdown" which confused me.

Comment (ScopeyNZ) [reply src/stories/FormFields.stories.mdx:8]: Yep I did. I've renamed the field name. I forgot to change it back after I was making sure the fields were consistent.

**Comment by ScopeyNZ** [comment]
Thanks. My simplification moved the click handler, which meant that clicks on the popover were causing it to toggle. I've fixed it by using a portal to hoist the popper to the body - this also fixes an issue where the popper wouldn't break out of overflow containers: https://chec-ui.netlify.app/?path=/story/form-fields-dropdown--within-a-chec-modal-popping-out-of-overflow .

File: src/stories/FormFields.stories.mdx (hunk 1 above)
Comment (robbieaverill) [reply src/stories/FormFields.stories.mdx:8]: The text field label should still be updated

**Pull request merged**